packages/collector/test/apps/agentStubControls.js

```diff
@@ -43,6 +43,9 @@ class AgentStubControls {
         env.KAFKA_TRACE_CORRELATION = opts.kafkaConfig.traceCorrelation.toString();
       }
     }
+    if (opts.ignoreEndpoints) {
+      env.INSTANA_IGNORE_ENDPOINTS = JSON.stringify(opts.ignoreEndpoints);
```

Comment:
I am testing the agent configuration for `ignoreEndpoints` and passing the config to `agentControls` during the tests. You can refer to the implementation [here](https://github.com/instana/nodejs/pull/1448/files#diff-804b7e039eceb0b7919638c73fd0123549361680fd0283aa0ffa654311c08913). Similar to other related tests for agent, the configuration is assigned to an environment variable so that it is available in the agent stub.